integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -568,6 +572,48 @@ def save(self, *args: Any, **kwargs: Any) -> None:
             self.last_updated = timezone.now()
         super().save(*args, **kwargs)
 
+    @transaction.atomic
+    def cleanup_autosaves(self, keep=3):
+        """
+        Delete all but the last ``keep`` AUTO_SAVEs
+        and renumber all affected versions to be continuous.
+
+        :param keep: How many auto saves should be kept
+        :type keep: int
+        """
+        logger.debug("Cleaning up autosaves except the latest %s ones.", keep)
+        delete_auto_saves = self.foreign_object.translations.filter(
+            language=self.language, status=status.AUTO_SAVE
+        )[keep:]
+        if not delete_auto_saves:
+            logger.debug("There are at most %s auto saves, nothing to clean up.", keep)
+            return
+
+        logger.debug("Deleting autosaves: %r", delete_auto_saves)
+        first_deleted_version = list(
+            delete_auto_saves.values_list("version", flat=True)
+        )[-1]
+        delete_auto_save_ids = delete_auto_saves.values_list("id", flat=True)
+        self.foreign_object.translations.filter(id__in=delete_auto_save_ids).delete()
```

Comment:
```suggestion
        first_deleted_version = delete_auto_saves[-1].version
        self.foreign_object.translations.filter(
            id__in=[t.id for t in delete_auto_saves]
        ).delete()
```
I don't see a reason to make two additional queries just to fetch objects we already have. Converting the queryset to a list allows us to re-use the previous results.